src/widgetastic/browser.py

```diff
@@ -751,6 +751,15 @@ def send_keys_to_focused_element(self, *keys):
         """
         ActionChains(self.selenium).send_keys(*keys).perform()
 
+    def copy(self):
```

Comment:
Looks good @tpapaioa, can you add to the unit test suite so we have coverage for this? A simple text field to select+copy, and a text input to paste it into and read for validation should do it.